src/MeadeCommandProcessor.cpp

```diff
@@ -1669,10 +1691,24 @@ String MeadeCommandProcessor::handleMeadeExtraCommands(String inCmd)
 #if (WIFI_ENABLED == 1)
             return wifiControl.getStatus() + "#";
 #endif
-
             return "0,#";
         }
+        else if (inCmd[1] == 'h')  // :XGh#
+        {
+#if (DEW_HEATER == 1)
+            char scratchBuffer[10];
+            sprintf(scratchBuffer, "%d,%d#", _mount->getHeater(0), _mount->getHeater(1));
+            return String(scratchBuffer);
+#endif
+            return "0,0#";
+        }
     }
+    else if (inCmd[0] == 'R')
```

Comment:
Please add an entry to the docs area above, detailing this command.

Reply:
changed to XRST and doc'd